text-encode-transform.js

```diff
@@ -96,8 +64,8 @@
                           });
   }
 
-  addReadableAndWritable(self.TextEncoder.prototype, encoder => {
-    return new TransformStream(new TextEncodeTransformer(encoder));
+  addReadableAndWritable(self.TextEncoder.prototype, () => {
```

Comment:
If you want to remove this parameter, you may want to remove the parameter in TextEncodeTransfomer constructor (I'm not sure if leaving the parameter is better or not, but I'm uncomfortable with the inconsistency among here, TextEncodeTransformer constructor and the call-site in addReadableAndWritable) .

Reply:
I removed it, thanks.